uk/guide/migrating-4.md

```diff
@@ -419,7 +419,7 @@ Running the above `npm` command will update `package.json` as follows:
     "errorhandler": "^1.1.1",
     "express": "^4.8.0",
     "express-session": "^1.7.2",
-    "jade": "^1.5.0",
+    "pug": "^1.5.0",
```

Comment:
> But what is your advice for older versions of jade for instance 2x because i just went & changed all jade to pug

I don't understand your question, I'm sorry.